### ISSUE TITLE
get_stream_playback() of AudioStreamPlayer always returns null.

### ISSUE THREAD

**Issue by kobligo** [godotengine/godot#72048]
### Godot version

v4.0.beta15.official [4fa6edc88]

### System information

Windows 10, Vulkan, RTX3080

### Issue description

The get_stream_playback() function introduced in beta15 (#71855) always returns null instead of an object of type AudioStreamPlaybackPolyphonic.

### Steps to reproduce

Run the included sample project and look at the console.

### Minimal reproduction project

[PlaybackBug.zip](https://github.com/godotengine/godot/files/10500727/PlaybackBug.zip)

**Comment by the-sink** [comment]
Also encountering this issue on my project, it returns null along with the error
```
E 0:00:00:0949   _ready: Player is inactive. Call play() before requesting get_stream_playback().
  <C++ Error>    Condition "stream_playbacks.is_empty()" is true. Returning: Ref<AudioStreamPlayback>()
  <C++ Source>   scene/3d/audio_stream_player_3d.cpp:787 @ get_stream_playback()
```

Following the advice of the error, I tried running ``play()`` immediately before ``get_stream_playback()``, to no avail. Although, waiting a frame before getting stream playback does appear to work, i.e.:

```gdscript
stream_player.play()
await get_tree().process_frame
var playback = stream_player.get_stream_playback() as AudioStreamPlaybackPolyphonic
```

The original code sample provided in #71855 doesn't seem to be working anymore due to this.

**Comment by TokageItLab** [comment]
In AudioStreamPlayer, audio_stream_playback is activated immediately after play(), but in AudioStreamPlayer2D and 3D, it is activated in the next physics_process(), so an empty reference was returned immediately after play(). It is fixed by #72233.
